e2etest/pkg/frr/container/container.go

```diff
@@ -36,15 +37,23 @@ type FRR struct {
 	Ipv6           string
 }
 
+type Config struct {
```

Comment:
This seems pretty similar to the containerConfig struct we already have. WDYT about combining them here and have the Start func receive only that?

Reply:
Not sure I got it right. containerConfig is a wrapper to nc / rc (and now cc).
I initially thought to move it here (and have `Start` accepting it, as you said), but I wasn't sure why it wasn't done in the first place, so I kept it consistent.
I'll move it.